application/testing/CMakeLists.txt

```diff
@@ -695,6 +695,8 @@ f3d_test(NAME TestInteractionGroupGeometriesLoadParentDirectory DATA mb/mb_0_0.v
 f3d_test(NAME TestInteractionInvertZoom DATA suzanne.ply ARGS --invert-zoom INTERACTION DEFAULT_LIGHTS)
 f3d_test(NAME TestInteractionCameraHotkeys DATA cow.vtp INTERACTION DEFAULT_LIGHTS)
 f3d_test(NAME TestInteractionZoomToMouse DATA cow.vtp INTERACTION DEFAULT_LIGHTS)
+f3d_test(NAME TestInteractionOrthographicProjection DATA cow.vtp INTERACTION DEFAULT_LIGHTS)
+f3d_test(NAME TestInteractionZoomToggleOrthographicProjection DATA cow.vtp INTERACTION DEFAULT_LIGHTS)
```

Comment:
which one of the test is pressing 5 two times ?

Reply:
TestInteractionZoomToggleOrthographicProjection

It does a zoom out and then presses 5 and then 5 again. I think that is wrong, it need to be

Press 5
Zoom out
Press 5

Reply:
The updated test case performs the following

     1. Zooms out by 1 mouse scroll wheel rotation
     2. Presses "5" to enable orthographic projection
     3. Zooms out a further 5 rotations of the scroll wheel
     4. Presses "5" to disable orthographic projection
 
The test case was implemented in this way so that it tests both the toggle mode in conjunction with the zoom mode.

I also did not move the mouse during the test so there are no unnecessary events (as before)